src/Microsoft.Framework.PackageManager/List/DependencyListCommand.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Versioning;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+
+namespace Microsoft.Framework.PackageManager.List
+{
+    internal class DependencyListCommand
+    {
+        private readonly DependencyListOptions _options;
+
+        public static void Initialize(CommandLineApplication c, Reports reports)
+        {
+            c.Description = "Print the dependencies of a given project.";
+            var runtimeFolder = c.Option("--runtime",
+                "A path to the folder contains framework assemblies. The framework projects usually depend on assemblies in this folder. The command will try to find if there is an assembly in this folder bear the same name to resolve dependencies.",
+                CommandOptionType.SingleValue);
+            var local = c.Option("-l|--local",
+                "Show the local assembly files been depended on.",
+                CommandOptionType.NoValue);
+            var tree = c.Option("-t|--tree",
+                "Show the dependencies in tree mode",
+                CommandOptionType.NoValue);
+            var treeLevel = c.Option("--tree-level",
+                "The level of tree to expand in tree mode.",
```

Comment:
Should also rename related properties/types/params from `treeLevel` to `maxTreeDepth`.